### PR TITLE
Move Validator from PayumBundle

### PR DESCRIPTION
**Pull request by pantelm**
As requested

### CODE REVIEW

**Comment by makasim** [comment]
Needs some tests

**Comment by pantelm** [comment]
I have some trouble with testing the validator. I hope it will do the job.

File: composer.json [hunk 1]
```diff
@@ -46,7 +46,8 @@
         "fp/klarna-invoice": "0.1.*",
         "stripe/stripe-php": "~1.0",
         "ext-soap": "*",
-        "twig/twig": "~1.0"
+        "twig/twig": "~1.0",
+        "symfony/validator": "~2.6"
```
Comment (makasim) [inline composer.json]: put it to require-dev

**Comment by makasim** [comment]
The tests looks good!

**Comment by makasim** [comment]
Thanks.

**Comment by makasim** [comment]
Also back ported to stable 0.14 branch, https://github.com/Payum/Payum/releases/tag/0.14.4

**Pull request merged**